dateutil/parser.py

```diff
@@ -911,47 +910,47 @@ def _parse(self, timestr, dayfirst=None, yearfirst=None, fuzzy=False,
                     # "my time +3 is GMT". If found, we reverse the
                     # logic so that timezone parsing code will get it
                     # right.
-                    if i+1 < len_l and l[i+1] in ('+', '-'):
-                        l[i+1] = ('+', '-')[l[i+1] == '+']
+                    if i + 1 < len_l and l[i + 1] in ('+', '-'):
+                        l[i + 1] = ('+', '-')[l[i + 1] == '+']
```

Comment:
Not strictly pertinent, but in a perfect world we should avoid altering the tokens.